documentation/modules/proc-iot-installing-services.adoc

```diff
@@ -30,10 +30,20 @@ ifeval::["{cmdcli}" == "oc"]
 ----
 ./install/components/iot/examples/k8s-tls/create
 {cmdcli} create secret tls iot-mqtt-adapter-tls --key=install/components/iot/examples/k8s-tls/build/iot-mqtt-adapter-key.pem --cert=install/components/iot/examples/k8s-tls/build/iot-mqtt-adapter-fullchain.pem
-{cmdcli} create secret generic iot-service-ca --from-file=service-ca.crt=install/components/iot/examples/k8s-tls/build/trusted-certs.pem
 ----
 endif::[]
 
++
+[NOTE]
+====
+If your cluster is not running on the `localhost`, you need to specify cluster IP address when creating certificates. For example:
```

Comment:
You actually need to specify the hostname, but you can use "nip.io" if you don't have one, with your IP address.

I know that, in this case, it only is a small difference, but putting your actual IP here wouldn't work if I understood it right.